Advertise EPSchedule Club

ajosan25 opened the pull request.

juberti commented on static/css/index.css:
```diff
@@ -36,6 +36,30 @@ neon-animated-pages {
+  width: 600px;
+}
+
+.joinbutton {
```
suggest using similar conventions here, e.g., .join-button, .github-button

juberti commented on templates/index.html:
```diff
@@ -111,15 +111,16 @@
               </paper-icon-button>
               <span class="title" id="day">Today</span>
-
+              <div style = "position:relative; left:-20px; background-color:yellow;">
```
do you need a wrapping div here?

juberti commented on templates/index.html:
```diff
@@ -209,6 +210,19 @@ <h1>
   </paper-dialog>
 
+  <paper-dialog id="join-epschedule" class="join-dialog">
+    <div id="GithubUpdateBox">
```
did you mean to use this id?

juberti commented on templates/index.html:
```diff
@@ -209,6 +210,19 @@ <h1>
+      <p>We meet every Thursday at 11:10 in TMAC-007.</p>
+      <p>If you have any suggestions for potential features, please click the three dots in the left panel and click "Report a bug".</p>
+      <a href = "https://forms.microsoft.com/r/sEqxAVt4PB">Join Us</a>
+      <p> Check us out: </p>
```
```suggestion
      <p>Check us out:</p>
```

juberti commented on templates/index.html:
```diff
@@ -209,6 +210,19 @@ <h1>
+      <p>Click "About" for more details.</p>
+      <p>We meet every Thursday at 11:10 in TMAC-007.</p>
+      <p>If you have any suggestions for potential features, please click the three dots in the left panel and click "Report a bug".</p>
+      <a href = "https://forms.microsoft.com/r/sEqxAVt4PB">Join Us</a>
```
```suggestion
      <a href="https://forms.microsoft.com/r/sEqxAVt4PB">Join Us</a>
```

juberti commented on templates/index.html:
```diff
@@ -209,6 +210,19 @@ <h1>
+      <p>If you have any suggestions for potential features, please click the three dots in the left panel and click "Report a bug".</p>
+      <a href = "https://forms.microsoft.com/r/sEqxAVt4PB">Join Us</a>
+      <p> Check us out: </p>
+      <button onclick="window.location.href = 'https://github.com/EastsidePreparatorySchool/epschedule'" class="githubbutton"><img src="../static/images/github.svg"></button>
```
should this just be an &lt;a&gt; element rather than a button? (same for the button below)

juberti commented on static/js/index.js:
```diff
@@ -61,7 +61,10 @@ function reportBug() {
   window.open("https://github.com/EastsidePreparatorySchool/epschedule/issues");
 }
-
+function collegeAdmission(){
```
humorous but maybe just call this `joinEpschedule`?

ajosan25 commented:
I completely revamped the feature. Essentially, I've removed the box that pops up (with all the info), and just changed it to a direct Microsoft Forms link. They can find the information easier there, and it's less of a hassle. Sorry for the confusion.

We'll hold off on merging until August

ajosan25 commented:
Let's merge this one as well @juberti. Changed it from EPSchedule Club to Hack Club.

juberti commented:
are the SVGs that you added used in this PR?

ajosan25 commented:
Removed all of the unused svgs
